src/java-interop/java-interop.mdproj

```diff
@@ -94,16 +88,36 @@
     </ItemGroup>
     <PropertyGroup>
       <_CppFlags>@(_Defines -&gt; '-D%(Identity)', ' ')</_CppFlags>
-      <_Includes>@(Include-&gt;'-I %(Identity)', ' ') @(JdkIncludePath-&gt;'-I %(Identity)', ' ')</_Includes>
+      <_Includes>@(MonoIncludePath-&gt;'-I %(Identity)', ' ') @(JdkIncludePath-&gt;'-I %(Identity)', ' ')</_Includes>
       <_LinkFlags>-fvisibility=hidden -Wl,-undefined -Wl,suppress -Wl,-flat_namespace</_LinkFlags>
-      <_Libs>-L /Library/Frameworks/Mono.framework/Libraries -lmonosgen-2.0 </_Libs>
+      <_Libs>$(MONO_LIBS)</_Libs>

```

Comment:
This would need to be `$(MonoLibs)`, because that's the MSBuild property that is created in `MonoInfo.props`.